Crashlytics/Crashlytics/Models/FIRCLSFileManager.m

```diff
@@ -124,6 +126,15 @@ - (BOOL)moveItemAtPath:(NSString *)path toDirectory:(NSString *)destDir {
   return YES;
 }
 
+- (BOOL)didCrashOnPreviousExecution {
+  NSString *crashedMarkerFileName = [NSString stringWithUTF8String:FIRCLSCrashedMarkerFileName];
+  NSString *crashedMarkerFileFullPath =
+      [[self rootPath] stringByAppendingPathComponent:crashedMarkerFileName];
+  self.didCrashPreviously =
+      self.didCrashPreviously || [self fileExistsAtPath:crashedMarkerFileFullPath];
```

Comment:
If you want to memoize this you can use dispatch_once eg: https://github.com/firebase/firebase-ios-sdk/blob/0d73868de6df07272705d98c98950e8f82b8601b/Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m#L386-L389